apps/src/music/views/TimelineElement.tsx

```diff
@@ -60,8 +60,13 @@ const TimelineElement: React.FunctionComponent<TimelineElementProps> = ({
     ? eventData.instrumentType
     : 'beat';
 
+  // The format of an id is "soundType/soundName" so parsing out the soundName
```

Comment:
Just for interest's sake: it's sometimes `soundType` but sometimes just a `folder`, especially for sounds in packs.